### DISCUSSION TITLE
Creating new tags?

### DISCUSSION THREAD

**Discussion by 2br-2b** [KevinBatdorf/code-block-pro#321] [Q&A] [answered]
Hello! I recently found this plugin and it's really nice! It's really helping format my code blocks on my blog.

I noticed, however, that my posts with Code Block Pro blocks are all getting given many new hex color tags ([example](https://codegito.xyz/tag/d8dee9/)). I tried deleting the tag, but the tag seems to have recreated and reassigned itself. Is this Code Block Pro's doing? If so, why, and is there any way do disable this?

**Comment by KevinBatdorf** [comment, marked as answer]
Hey, I checked your example page but maybe you solved it? I'm guessing you have another plugin that attempts to pull out hashtags to create post tags? If so, you should ask them how to "allow list" color tags so they don't get used.

**Reply by 2br-2b** [reply]
I don't have a plugin that explicitly does this, but it's probably bundled in with another plugin I have - time to troubleshoot 😅

**Reply by KevinBatdorf** [reply]
That's WordPress for you 😅 let me know what you find out and if you get stuck somewhere.

**Reply by 2br-2b** [reply]
I think that's it - activity pub has this option:

![IMG_20240530_220515](https://github.com/KevinBatdorf/code-block-pro/assets/43968991/4a8b0911-e9c0-47ee-aac4-46bf01ea3e64)

And that's probably generating the hashtags. Thanks for the help!
